packages/nc-gui/composables/useLTARStore.ts

```
@@ -482,39 +496,54 @@
-            {
-              limit: String(limit ?? childrenListPagination.size),
-              offset: String(offset),
-              where:
-                childrenListPagination.query && `(${relatedTableDisplayValueProp.value},like,${childrenListPagination.query})`,
-            } as any,
-          )
+          const where = childrenListPagination.query
+            ? `(${relatedTableDisplayValueProp.value},${
+                isDateOrDateTimeCol(relatedTableDisplayValueColumn.value!) ? 'eq,exactDate' : 'like'
+              },${childrenListPagination.query})`
+            : undefined
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)